cpp/src/jni/jni_wrapper.cc

```diff
@@ -467,6 +441,11 @@ Java_io_glutenproject_vectorized_NativeColumnarToRowJniWrapper_nativeConvertColu
   jobject native_columnar_to_row_info = env->NewObject(
       native_columnar_to_row_info_class, native_columnar_to_row_info_constructor,
       instanceID, offsets_arr, lengths_arr, address);
+  struct ArrowArray out_array;
+  struct ArrowSchema out_schema;
+  gluten::JniAssertOkOrThrow(arrow::ExportRecordBatch(*rb, &out_array, &out_schema));
+  ArrowArrayRelease(&out_array);
+  ArrowSchemaRelease(&out_schema);
```

Comment:
It seems like we close `out_array` `out_schema` immediately. What's the purpose of doing this?

Reply:
OK, I have removed this